brianlong/www.validators.app
# 182548554 fix 33 stake badge

#### What's this PR do?
- Fixes 33_stake badge on validators index view

#### How should this be manually tested?
- Run daemons for mainnet (gather daemon + score v1 daemon)
- Check the list on homepage: 
  - if you sort validators by **score**, badge should not be visible at all on any page
  - if you sort validators by **stake**, the badge should be visible on the correct page (currently after the 26th validator), and not on any other page
  - if you fillter by favourite, the badge should not be displayed
  - if you use search, the badge should not be displayed

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/n/projects/2177859/stories/182548554)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
